beacon-chain/core/altair/epoch_spec.go

```diff
@@ -35,7 +36,7 @@ func ProcessSyncCommitteeUpdates(ctx context.Context, beaconState state.BeaconSt
 			return nil, err
 		}
 		if err := helpers.UpdateSyncCommitteeCache(beaconState); err != nil {
-			return nil, err
+			log.Errorf("Could not update sync committee cache: %v", err)
```

Comment:
Please use `WithError`

```suggestion
			log.WithError(err).Error("Could not update sync committee cache")
```